### DISCUSSION TITLE
call to FundMe.getVersion errored: execution reverted

### DISCUSSION THREAD

**Discussion by wuyiNLP** [smartcontractkit/full-blockchain-solidity-course-py#3] [Q&A] [answered]
I followed to the Lesson 3, starting to use Oracle. But even I copied the code, it gave feedback like:
 call to FundMe.getVersion errored: execution reverted
It seems it failed to refer the ETH price from Chainlink. How to make it correct

**Comment by PatrickAlphaC** [comment]
Thanks for this question! Did you deploy the code to the rinkeby testnet?

**Reply by wuyiNLP** [reply]
> Thanks for this question! Did you deploy the code to the rinkeby testnet?

I changed rinkeby to kovan and it worked!

**Reply by PatrickAlphaC** [reply]
Wait... it worked on Kovan? It should work on rinkeby...

**Reply by wuyiNLP** [reply]
> Wait... it worked on Kovan? It should work on rinkeby...
The chainlink gives the example like this. I tried to check the ETH/USD address by myself under rinkeby, but it doesn't work. So I instead use the address under kovan, and it works! (of course I changed Metamask to Kovan as well...
/**
     * Network: Kovan
     * Aggregator: ETH/USD
     * Address: 0x9326BFA02ADD2366b30bacB125260Af641031331
     */

**Reply by PatrickAlphaC** [reply]
Ah, I see. 

Lesson 3 code works with rinkeby though no? And thanks for the feedback and making this conversation!

**Reply by PatrickAlphaC** [reply]
aaahhhhhhhh I must have changed around... Thanks for updating!

So could you see what the timestamp and issue resolution is? So at 3:08:00 I add the wrong address, and the resolution is to use the correct rinkeby address?

**Reply by AlbertoSinigaglia** [reply]
I'm sorry, I double checked, and turns out that is just a combination between the fact that the first Remix that you use (provided by the chainlink) documentation is Kovan based, so if someone followed you in the test of "getting the latest price" from the documentation, copying the address from the documentation won't work (since the address is for Rinkeby network but the example is on the kovan)... 
Also might happen to find the Kovan address before the Rinkeby address in the documentation... 

So there is no "patch" to do, just maybe a warning to use the address from the correct network, but as mentioned, it's easily solvable since you shoed how to get eth on Kovan

**Reply by PatrickAlphaC** [reply]
aahhhh ok ok... I'll make an update here to make sure that's clear.

**Reply by manavnotnani** [reply]
thanks for asking this question. It really helped

**Comment by PatrickAlphaC** [comment, marked as answer]
I have added a `lesson 3` section to the issues tracker, thank you!

https://github.com/smartcontractkit/full-blockchain-solidity-course-py/blob/main/chronological-issues-from-video.md#lesson-3
